add steps for update the dependency and repo (#3257)
istio/istio.io: content/docs/setup/kubernetes/minimal-install/index.md

## Prerequisites

> Refer to the [prerequisites](/docs/setup/kubernetes/quick-start/#prerequisites) described in the Quick Start guide.  The following commands have relative references in the Istio directory. You must execute the commands in Istio's root directory.

Reviewer: Please be careful how you reference other documents. This link is difficult to maintain because you place the document and the section of the document far apart in the text and the document isn't even part of the link. If the title of the document changes, someone will have to find the reference in the text. also be mindful of the 80 character line length to make reviewing easier and for consistency.
```suggestion
Refer to the [Kubernetes Installation prerequisites](/docs/setup/kubernetes/quick-start/#prerequisites) for more information.  The following commands have relative references in the Istio directory. You must execute the commands in Istio's root directory.
```

    {{< tip >}}
    To customize Istio and install addons, use the `--set <key>=<value>` option in the helm template or install command. [Installation Options](/docs/reference/config/installation-options/) references supported installation key and value pairs.
    {{< /tip >}}
    
## Installation steps

> The following commands have relative references in the Istio directory. You must execute the commands in Istio's root directory.

Reviewer: This text is a duplicated from above, please remove.
```suggestion
```

Choose one of the following two **mutually exclusive** options described below.

> - To use Kubernetes manifests to deploy Istio, follow the instructions for [option #1](#option-1).		

Reviewer: What is the key differentiation between the options? I could argue that option 2 also uses manifests, albeit through Tiller.
To help readers avoid confusion, be explicit about what one option needs that the other doesn't, in this case Tiller.
```suggestion
- To deploy Istio without using Tiller, follow the instructions for [option #1](#option-1).		
```

## Installation steps

> The following commands have relative references in the Istio directory. You must execute the commands in Istio's root directory.

Reviewer: Consider adding this note under prerequisites.

Author: moved!

> 1.  Update Helm's dependencies:

Reviewer: Is this needed for both options? If so, please include it as a prerequisite. Otherwise, please include this step at the beginning of the corresponding option.

    {{< text bash >}}
    $ helm repo add istio.io "https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/release-1.1-latest-daily/charts/"

>     $ helm dep update install/kubernetes/helm/istio

Reviewer 2: `helm dep update` is no longer needed.

Author: so true! removed

    {{< /text >}}

> 1. Choose one of the following two **mutually exclusive** options described below.

Reviewer: Please do not remove the list explaining what the two options are. The list provides guidance to the readers to skip to the option they need without having to scroll down.

    {{< tip >}}

>     To customize Istio and install addons, use the `--set <key>=<value>` option in the helm template or install command. [Installation Options](/docs/reference/config/installation-options/) references supported installation key and value pairs.

Reviewer: This tip would be better placed as an additional paragraph before the Prerequisites section since it points readers to additional information regarding the available options.
